packages/engine/Source/Core/ArcGISTiledElevationTerrainProvider.js

```diff
@@ -446,7 +446,7 @@ ArcGISTiledElevationTerrainProvider.prototype.requestTileGeometry = function (
         encoding: that._encoding,
       });
     })
-    .catch(function (error) {
+    .catch(async function (error) {
```

Comment:
Sorry, this is technically an unrelated change, but I was able to track down what I believe was causing https://github.com/CesiumGS/cesium/issues/11231 while troubleshooting unit tests.